### PR TITLE
netbird: new package

### PR DESCRIPTION
**Pull request by oskarirauta**
NetBird is an open-source VPN management platform built on top of WireGuard® making it easy to create secure private networks for your organization or home. It requires zero configuration effort leaving behind the hassle of opening ports, complex firewall rules, VPN gateways, and so forth.

Maintainer: me / @oskarirauta
Compile tested: x86_64, recent git
Run tested: x86_64, recent git

Description: Netbird is similar vpn service as tailscale and zerotier.
Signed-off-by: Oskari Rauta <oskari.rauta@gmail.com>

### CODE REVIEW

**Comment by oskarirauta** [comment]
@1715173329 

another competitor for tailscale and zerotier.

File: net/netbird/Makefile [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  secure private networks for your organization or home.
+
+  It requires zero configuration effort leaving behind the hassle of opening ports, complex firewall rules, VPN 
+  gateways, and so forth.
+endef
+
+define Package/netbird/install
+	$(call GoPackage/Package/Install/Bin,$(PKG_INSTALL_DIR))
+	$(INSTALL_DIR) $(1)/usr/bin $(1)/etc/init.d
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/client $(1)/usr/bin/
```
Comment (1715173329) [inline net/netbird/Makefile:47]:
```suggestion
	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/client $(1)/usr/bin/netbird
```

or maybe `netbird-client`?

**Review by 1715173329 — CHANGES_REQUESTED** [review]

File: net/netbird/Makefile [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_BUILD_DEPENDS:=golang/host
+PKG_BUILD_PARALLEL:=1
+PKG_USE_MIPS16:=0
+
+GO_PKG:=github.com/netbirdio/netbird
+GO_PKG_BUILD_PKG:=github.com/netbirdio/netbird/client
+GO_PKG_LDFLAGS_X:=github.com/netbirdio/netbird/client/system.version=$(PKG_VERSION)
+GO_PKG_LDFLAGS:=-s -w
```
[the remark this answers is not in the document]
Comment (oskarirauta) [reply net/netbird/Makefile]: And you said you aren't good with go.... ;) - Obviously that's under-statement :)

File: net/netbird/Makefile (hunk 1 above)
Comment (oskarirauta) [reply net/netbird/Makefile]: No, it's both server and client, so netbird- just my bad that I mistakenly forgot to do it like that- on my test environment it was like that, but I do git pushes from other path and sometimes all stuff isn't in sync.. So suggestion is correct.

**Comment by oskarirauta** [comment]
Good pointers. I made the requested changes and accepted them as-is or as in proposals they were.

**Comment by oskarirauta** [comment]
Something happened in tests.... x86_64 target..

File: net/netbird/Makefile (hunk 2 above)
[the remark this answers is not in the document]
Comment (1715173329) [reply net/netbird/Makefile]: > And you said you aren't good with go.... ;) - Obviously that's under-statement :)

really, but i know the makefile rules :p

**Comment by 1715173329** [comment]
> @1715173329
> 
> Something happened in tests.... x86_64 target.. Can you check it..?

github actions sucks

and, go_pkg was removed by mistake, please re-add it.

**Review by 1715173329 — APPROVED** [review]

**Comment by dangowrt** [comment]
> another competitor for tailscale and zerotier.

... and OpenWrt's `unetd` which also does practically the same.

**Comment by oskarirauta** [comment]
@dangowrt - but unetd relies on your own server unlike zerotier, netbird and tailscale - or did I understant it all wrong? It uses zerotier mesh protocol just like tailscale and is very similar in many ways, except all open source - when tailscale is only partially, but it would be much more popular if they would have mobile device (iOS, android, etc) support as well - it's been requested, and planned - but last post seems to be a year old and no mentions about it after that, so....

**Pull request merged**